test/IISIntegration.FunctionalTests/OutOfProcess/HttpsTest.cs

```diff
@@ -60,19 +60,19 @@ public Task Https_HelloWorld_CoreCLR_X64_Portable(string ancmVersion)
                     SiteName = "HttpsTestSite", // This is configured in the Https.config
                     TargetFramework = runtimeFlavor == RuntimeFlavor.Clr ? "net461" : "netcoreapp2.0",
                     ApplicationType = applicationType,
+                    ANCMVersion = ancmVersion,
                     Configuration =
 #if DEBUG
```

Comment:
You don't need this DEBUG switch anymore, the depoyer should take care of that for you.

Reply:
Which commit fix this in hosting?

Reply:
https://github.com/aspnet/Hosting/commit/9d82942a1a4650da9682b358262de5635a537f3f#diff-8478a3aebbec4604e8f2cf79bb152195R53